plugins/woocommerce-admin/client/launch-your-store/hub/styles.scss

```
@@ -2,6 +2,11 @@
 	min-width: 600px;
 	overflow: auto;
 
+	&.woocommerce-launch-your-store-success {
+		overflow-x: clip;
+		min-width: none;
```

It seems the `min-width: none` is [not a valid rule](https://github.com/woocommerce/woocommerce/assets/3747241/074e50f9-cbfe-4cb9-8ede-7ff19a3128cf).

```suggestion
		min-width: unset;
```

---

Thanks! Sharp eyes! I blame copilot 🙈